stdlib/public/Darwin/Foundation/NSStringAPI.swift

```diff
@@ -178,6 +178,10 @@ extension String {
   /// Creates a string by copying the data from a given
   /// C array of UTF8-encoded bytes.
   public init?(utf8String bytes: UnsafePointer<CChar>) {
+    if let str = String(validatingUTF8: bytes) {
+      self = str
+      return
+    }
     if let ns = NSString(utf8String: bytes) {
```

Comment:
Why not remove the NSString-and-bridge part of this entirely?
Then we could also make it `inlineable` since it trivially forwards to the stdlib function.

Reply:
I'm trying to preserve behavior if the bytes are invalidly encoded. We should probably deprecate this initializer.